### PR TITLE
MooseEnum error

### PR DESCRIPTION
**Pull request by micahjohnson150**
@aeslaughter this was the error I was telling you about sometime ago. I got it figured out.

Attempting to set std::vector MooseEnum "execute on" when it already exists as a MultiMooseEnum. 

This error was discovered when trying to run 2D bubble problem. While there is a criteria test created , it is incomplete and not being used currently. This fix eliminated the error but the testing is still not in place.

### CODE REVIEW

File: include/PikaRevision.h [hunk 1]
```diff
@@ -0,0 +1,8 @@
+/* THIS FILE IS AUTOGENERATED - DO NOT EDIT */
```
Comment (aeslaughter) [inline include/PikaRevision.h]: Do not include this file in the commit.

**Comment by aeslaughter** [comment]
This is the correct fix. Create an issue and then reference the number in the commit message, this will then run through the testing system.

File: include/PikaRevision.h (hunk 1 above)
Comment (micahjohnson150) [inline include/PikaRevision.h]: What am I supposed to do with this file? Delete it? I don't remember this ever showing up before...

Comment (aeslaughter) [inline include/PikaRevision.h]: Just remove it from the commit and you can probably add it to the
.gitignore as well. We generate this file to keep track of what version of
MOOSE Pika is using.

Remove it by:
get reset -- include/PikaRevision.h

**Pull request closed without merging**